js/vue/meetupEvents.vue

```diff
@@ -0,0 +1,96 @@
+
+<template>
+  <div>
+    <div class="meetups">
+      <div class="meetup-event" v-for="event in events" :key="event.id">
+        <h3 class="meetup-heading"> {{ event.name }}</h3>
+        <hr>
+        <div class="meetup-description">
+          <div v-html="event.description"></div>
+          <div class="meetup-details">
+            <div class="meetup-date"> <b>When:</b> {{ event.parsed_date }} </div>
+            <div v-if="event.venue" class="meetup-location"> <b>Where:</b> {{ event.venue.name }} </div>
+            <div v-else> <b>Where:</b> TBD </div>
+            <div class="meetup-who"> <b>Who:</b> {{ event.yes_rsvp_count }} already going</div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+//import node module dependencies
+import jsonp from 'jsonp';
+
+//import utility dependencies
+import { parameterize, formatDate } from '../utils/component-utils.js';
+
+
+const DESIRED_HACK_NIGHTS = 4;
+const HACK_NIGHT = "[Everyone] Full Stack Hack Night";
+const EVENT_URL = "http://api.meetup.com/newhavenio/events?";
+const PARAMS = {
+  page: 20,
+  status: 'upcoming',
+}
+               
+export default {
+  data: function() {
+    return {
+      events: [],
+    };
+  },
+  methods: {
+    getEvents: function() {
+      var requestUrl = EVENT_URL + parameterize(PARAMS);
+      jsonp(requestUrl, null, (err, data) => {
+        var rawEvents = data.data;
+        var eventsArray = [];
+        var foundHacks = 0;
+        this.events = rawEvents.filter( function(event) {
+          if (event.name !== HACK_NIGHT) { return true; }
+          foundHacks++;
+          if (foundHacks < DESIRED_HACK_NIGHTS) { return true; }
+          return false;
+        } )
+        .map( function(event, index) {
+          event["parsed_date"] = formatDate(event.local_date, event.local_time); 
+          return event; 
+        } );
```

Comment:
For fun I want to experiment turning this into a computed property. Are you OK if it works and I send you  a patch?

Reply:
Sure. its usually better to use computed properties anyway, I just didn't think it was worth it to set it up such that.

The other options i was thinking about was using a method and just doing that mapping in the vue with something like...
```javasript
{{ formatDate(event.local_date, event.local_time) }}
```
you could do a similar process with computed, though I don't know how you would get the computer property to be able to format the dates inside of the event object. I'm happy to see it happen though just make PR.

Reply:
Here are my changes: https://gist.github.com/sukima/b5dabb1951de8a53707efba27bd5fe42